ikarus/finiteelements/mechanics/materials/linearelasticity.hh

```diff
@@ -60,13 +51,23 @@ struct LinearElasticityT : Material<LinearElasticityT<ST>>
   static constexpr bool stressAcceptsVoigt = Base::stressAcceptsVoigt;
   static constexpr bool moduliToVoigt      = Base::moduliToVoigt;
   static constexpr bool moduliAcceptsVoigt = Base::moduliAcceptsVoigt;
-
-  // This factor denotes the differences between the returned stresses and moduli
-  // and the passed strain. For neoHooke the inserted quantity is C, the right Cauchy-Green tensor.
-  // The function relation between the energy and the stresses is S = 2*\partial \psi(C)/ \partial C.
-  // This factor is a pre-factor, which is the difference to the actual derivative is written here.
```

Comment:
Why did you remove this comment? Anyway I suggest this comment is moved to the interface class together with a line
`static constexpr double derivativeFactor = MI::derivativeFactorImpl;`